app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
Add inheritance from the `ABC` class as well for `Elf`, because the `Elf` class does not implement all abstract methods from the `Player` class.